mlprimitives/timeseries.py

```diff
@@ -0,0 +1,63 @@
+import pandas as pd
+import numpy as np
+
+
+def create_window_sequences(df_timeseries, window_size):
+    """
+        Function that takes in a Pandas.DataFrame and a window_size then creates
+            output arrays that correspond to a timeseries sequence with window_size overlap.
+            The output arrays can be fed into a timeseries forecasting model.
+        Inputs:
+            df_timeseries (Pandas.DataFrame): a Pandas dataframe which has 'timestamp'
+                and 'value' columns, and is sorted based on timestamp. 
+                The timestamp column is in UNIX format (in seconds).
+            window_size (int): number of values that overlap to create the sequence.
+        Outputs:
+            x (numpy.ndarray): contains the time series sequenced data.
+            y (numpy.ndarray): acts as the label for the forecasting problem.
+            time (numpy.ndarray): the corresponding timestamps series.
+    """
+    X = []
+    Y = []
+    time = []
+    for i in range(len(df_timeseries) - window_size):
+        X.append(df_timeseries[i: i + window_size]['value'].values.copy().reshape([-1, 1]))
+        Y.append(df_timeseries[i + 1: i + window_size + 1]['value'].values.copy().reshape([-1, 1]))
+        time.append(df_timeseries.iloc[i + window_size]['timestamp'])
+    return np.asarray(X), np.asarray(Y), np.asarray(time)
```

Comment:
Would you mind adding a line break before this line? This is just to be consistent with the styling of the rest of the library.

Reply:
The Linter suggested I remove the line 😆

Reply:
That's strange. It should complain about two bank lines, but not about one.
Anyhow, let's try having one there and see it if accepts it, just to keep it consistent with the rest :-)